File: src/main/java/top/zbeboy/isy/glue/data/OrganizeGlue.java
package top.zbeboy.isy.glue.data;

import top.zbeboy.isy.glue.util.ResultUtils;
import top.zbeboy.isy.web.bean.data.organize.OrganizeBean;
import top.zbeboy.isy.web.util.DataTablesUtils;

import java.util.List;

/**
 * Created by lenovo on 2017-04-09.
 */
public interface OrganizeGlue {
    /**
     * 分页查询
     *
     * @param dataTablesUtils datatables工具类
     * @return 分页数据
     */
    ResultUtils<List<OrganizeBean>> findAllByPage(DataTablesUtils<OrganizeBean> dataTablesUtils);

    /**
     * 班级总数
     *
     * @return 总数
     */
    long countAll();
}

File: src/main/java/top/zbeboy/isy/glue/data/OrganizeGlue.kt
package top.zbeboy.isy.glue.data

import top.zbeboy.isy.glue.util.ResultUtils
import top.zbeboy.isy.web.bean.data.organize.OrganizeBean
import top.zbeboy.isy.web.util.DataTablesUtils

/**
 * Created by zbeboy 2017-12-03 .
 **/
interface OrganizeGlue {
    /**
     * 分页查询
     *
     * @param dataTablesUtils datatables工具类
     * @return 分页数据
     */
    fun findAllByPage(dataTablesUtils: DataTablesUtils<OrganizeBean>): ResultUtils<List<OrganizeBean>>

    /**
     * 班级总数
     *
     * @return 总数
     */
    fun countAll(): Long
}